search-service/src/test/kotlin/com/egm/stellio/search/util/QueryUtilsTests.kt

```diff
@@ -17,14 +17,40 @@ import java.time.ZonedDateTime
 @ActiveProfiles("test")
 class QueryUtilsTests {
 
+    @Test
+    fun `it should throw a BadRequestData exception if type or attrs are not present when it's mandatory`() {
```

Comment:
one of them is always mandatory

Reply:
not for get entity by id

Reply:
so maybe rename `when it's mandatory` to `when querying entites`